### PR TITLE
Add accept options to webhook

### PR DESCRIPTION
**Pull request by nikostsaganos**

### CODE REVIEW

File: source/localizable/smart_cart/webhook.html.md.erb [hunk 1]
```diff
@@ -326,7 +509,68 @@ Name           | Type   | Value | Description
     ],
     "created_at": "2019-10-29T10:39:23+02:00",
     "expires_at": "2019-10-29T16:39:23+02:00",
-    "dispatch_until": "2019-10-30T15:00:00+02:00"
+    "dispatch_until": "2019-10-30T15:00:00+02:00",
+    "accept_options": {
```
Comment (cvrac) [inline source/localizable/smart_cart/webhook.html.md.erb:513]: We shouldn't add `accept`/`reject` options for courier voucher creation payloads. 
The `state` attribute within the order payload should also be `accepted`, not `open`. Missed this in an earlier review :)

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 2]
```diff
@@ -38,3 +40,57 @@ Name                              | Type   | Value | Description
...
+`accept_options.pickup_window`    | Array  |             | [Pickup window](#accept-option-pickup-window)
+
+#### Accept option: Pickup location
+
+Valid pickup locations to post when
+[accepting a single order](/smart_cart/orders_api/#accept-a-single-order) (use `id`).
+
+Name                    | Type    | Description
+:---------------------: | :-----: | :-----------------------:
+`pickup_location.id`    | Integer | The location ID to use
```
Comment (cvrac) [inline source/localizable/smart_cart/_order_object.html.md.erb:63]: [super minor] How about adding `accept_options[].` / `reject_options[].` before each attribute name for pickup locations, pickup windows and rejection reasons? It's not that important, just to be somewhat consistent with the rest of the objects descriptions (see line item sizes). Feel free to ignore :)

**Comment by cvrac** [comment]
Let's also add `accepted` state value at https://github.com/skroutz/developer.skroutz.gr/blob/master/source/localizable/smart_cart/_order_object.html.md.erb#L6

File: source/localizable/smart_cart/_order_object.html.md.erb (hunk 2 above)
Comment (nikostsaganos) [reply source/localizable/smart_cart/_order_object.html.md.erb:63]: These are not arrays therefore, that's why I skipped the `[]` part... I think I'll leave them like this for now, wdyt?

File: source/localizable/smart_cart/webhook.html.md.erb (hunk 1 above)
Comment (nikostsaganos) [reply source/localizable/smart_cart/webhook.html.md.erb]: fixed thanks

**Pull request merged**